docs/usage/configuration-options.md

```diff
@@ -1556,6 +1556,18 @@ To adjust it down to 10s for all queries, do this:
 }
 ```
 
+### httpsCertificateAuthority
+
+`httpsCertificateAuthority` overrides the trusted CA certificate that defaults to the curated list of well-known CAs by Mozilla.
+
+### httpsPrivateKey
+
+Specifies the private key in PEM format for mTLS authentication. **Note: a Private Key MUST NOT be specified directly here to avoid losing confidentiality completely. Use [secrets](https://docs.renovatebot.com/self-hosted-configuration/#secrets) to pass it down securely instead.**
```

Comment:
```suggestion
Specifies the private key in PEM format for mTLS authentication.
**Note: a Private Key SHOULD NOT be specified directly here to avoid losing confidentiality completely.
Use [secrets](https://docs.renovatebot.com/self-hosted-configuration/#secrets) to pass it down securely instead.**
```